torchvision/models/detection/retinanet.py

```diff
@@ -614,3 +615,35 @@ def retinanet_resnet50_fpn(pretrained=False, progress=True,
         model.load_state_dict(state_dict)
         overwrite_eps(model, 0.0)
     return model
+
+
+def retinanet_mobilenet_v3_large_fpn(pretrained=False, progress=True,
+                                     num_classes=91, pretrained_backbone=True, **kwargs):
+    """
+    Constructs a RetinaNet model with a MobileNetV3-Large-FPN backbone. It works similarly
+    to RetinaNet with ResNet-50-FPN backbone. See `retinanet_resnet50_fpn` for more details.
+
+    Example::
+
+        >>> model = torchvision.models.detection.retinanet_mobilenet_v3_large_fpn(pretrained=True)
+        >>> model.eval()
+        >>> x = [torch.rand(3, 300, 400), torch.rand(3, 500, 400)]
+        >>> predictions = model(x)
+
+    Args:
+        pretrained (bool): If True, returns a model pre-trained on COCO train2017
+        progress (bool): If True, displays a progress bar of the download to stderr
+    """
+    if pretrained:
+        pretrained_backbone = False
+    backbone = mobilenet_fpn_backbone("mobilenet_v3_large", pretrained_backbone, returned_layers=[4, 5])
```

Comment:
We use the outputs of blocks C4 and C5 in our feature pyramid. 

On C5, we do the same as the paper and we use the layer just before pooling.

On C4, we deviate from the original paper that suggests using the "the **expansion** layer of the 13th bottleneck block". In our case we use the **output** of the 13th bottleneck because it's very hard to get the output of the expansion without completely refactoring the entire mobilenetV3 architecture. As a result our C4 feature output is 160x7x7 instead of 672x14x14. As a result both

This could lead to a faster model but might reduce the accuracy metrics. We'll do experiments to assess the difference.